docs/src/examples/components/Avatar/Variations/AvatarExampleImageCustomization.shorthand.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Avatar, Icon } from '@stardust-ui/react'
+
+const iconAsImage = (Image, props, children) => {
+  return <Icon name={'user'} circular variables={{ color: 'blue' }} styles={{ fontSize: '16px' }} />
```

Comment:
seems that the expression for `name` prop could be safely simplified to use just quotes